### PR TITLE
#139 Added localization

### PR DESCRIPTION
**Pull request by Chopikov**
- Added strings.xml
- Fixed some english strings

### CODE REVIEW

File: app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        изображения</string>
+    <string name="title_help_section_result">Результат</string>
+    <string name="help_section_load_image">Загрузка изображения из галереи и
+        получение фотографии с камеры.</string>
+    <string name="help_section_edit_image">Поворот, отражение и обрезка
+        изображения. Установка размеров кроссворда.</string>
+    <string name="help_section_result">Просмотр и сохранение
+        результата.</string>
+    <string name="help_info_load_image">Чтобы преобразовать изображение в
+        японский кроссворд, его нужно загрузить.</string>
```
Comment (maik-nack) [inline app/src/main/res/values-ru/strings.xml]: Logical warning

File: app/src/main/res/values-ru/strings.xml [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    <string name="title_help_beginning">Начало</string>
+    <string name="help_beginning_load_image">Чтобы загрузить изображение в
+        приложение, вам нужно нажать на кнопку \'ЗАГРУЗИТЬ КАРТИНКУ\' на
+        главном экране. По нажатию вы перейдете на экран, чтобы выбрать
+        изображение.</string>
+    <string name="title_help_image_picker">Выбор изображения</string>
+    <string name="help_image_picker">Все изображения на устройстве разложены по
+        папкам, в которых они находятся. Когда вы нажимаете на определенную
+        папку, появляются изображенияиз этой папки. Когда вы нажимаете на
```
Comment (maik-nack) [inline app/src/main/res/values-ru/strings.xml]: Lost space

File: app/src/main/res/values-ru/strings.xml [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    <string name="help_columns">Чтобы выбрать размер кроссворда, нажмите на
+        кнопку \'РАЗМЕР\' в правом нижнем углу.\n\nВы можете увидеть примерный
+        результат. Также вы можете увидеть над изображением текущий размер и
+        диапазон возможных размеров после преобразования. Около изображения есть
+        слайдер, который обеспечивает выбор количества столбцов. Число строк
+        рассчитывается автоматически. Выбранное изображение находится в центре
+        экрана, поверх него рисуется сетка. Чтобы зафиксировать свой выбор,
+        нажмите на галочку в правом верхнем углу.</string>
+    <string name="help_convert">После того, как вы настроили все, что вам нужно,
+        нажмите \'Конвертировать\' в правом верхнем углу. После нажатия появится
```
Comment (maik-nack) [inline app/src/main/res/values-ru/strings.xml]: Maybe it needs some words

File: app/src/main/res/values-ru/strings.xml [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+        У вас есть возможность нажать на стандартную кнопку \'назад\' или в
+        любое место мимо окна прогресса, чтобы отменить преобразование.</string>
+    <string name="help_info_result">После конвертации изображения в
+        японский кроссворд вы можете увидеть результат и сохранить его.</string>
+    <string name="help_save_thumbnail">Вы можете сохранить решение кроссворда.
+        Для этого нажмите на кнопку \'СОХРАНИТЬ МИНИАТЮРУ\' в левом нижнем углу.
+        Размер сохраненного изображения в пикселях равен размеру
+        кроссворда.</string>
+    <string name="help_save_nonogram">Вы можете сохранить результат в виде
+        нерешенного японского кроссворда. Для этого нажмите на кнопку
```
Comment (maik-nack) [inline app/src/main/res/values-ru/strings.xml]: `ё`?

File: app/src/main/res/values-ru/strings.xml [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+        \'СОХРАНИТЬ НОНОГРАММУ\' в правом нижнем углу.</string>
+    <string name="help_view_result">Вы можете увидеть полученный кроссворд.
+        В центре экрана решенный японский кроссворд. Его можно приблизить
```
Comment (maik-nack) [inline app/src/main/res/values-ru/strings.xml]: `ё`?

File: app/src/main/res/values-ru/strings.xml [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+    <string name="title_dialog_convert">Конвертация не удалась</string>
+    <string name="msg_convert">Конвертация не удалась.\n Вы можете попробовать
+        изменить размеры кроссворда или обрезать изображение.</string>
+
+    <string name="action_load_image">Загрузить Картинку</string>
+    <string name="action_open_about">О приложении</string>
+    <string name="action_open_help">Справка</string>
+
+    <string name="msg_description">Мобильное Android-приложение для конвертации
+        изображения в черно-белую нонограмму\n
```
Comment (maik-nack) [inline app/src/main/res/values-ru/strings.xml]: `ё`?

File: app/src/main/res/values-ru/strings.xml [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+        - Выберите изображение из галереи или сделайте фотографию\n
+        - Вы можете обрезать и повернуть выбранное изображение\n
+        - Выберите размер неграммы\n
```
Comment (maik-nack) [inline app/src/main/res/values-ru/strings.xml]: What is `неграммы`?

File: app/src/main/res/values-ru/strings.xml [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+    <string name="msg_process_image">Обработка изображения…</string>
+    <string name="action_change">Изменить</string>
+
+    <string name="title_result_activity">Результат</string>
+    <string name="action_save_thumbnail">Сохранить миниатюру</string>
+    <string name="action_save_nonogram">Сохранить нонограмму</string>
+    <string name="action_home_return">Домой</string>
+    <string name="title_dialog_home_return">Вернуться домой</string>
+    <string name="msg_home_return">Вы уверены, что хотите вернуться домой?
+        Нонограмма будет потеряна, если вы ее не сохраните.</string>
```
Comment (maik-nack) [inline app/src/main/res/values-ru/strings.xml]: `ё`?

Comment (maik-nack) [inline app/src/main/res/values-ru/strings.xml]: Maybe it needs past time

File: app/src/main/res/values-ru/strings.xml [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+    <string name="msg_home_return">Вы уверены, что хотите вернуться домой?
+        Кроссворд будет потерян, если вы ее не сохраните.</string>
+    <string name="action_return">ВЕРНУТЬСЯ</string>
+    <string name="action_cancel">ОТМЕНА</string>
+    <string name="msg_save_image_done">Изображение сохранено:</string>
+    <string name="msg_save_image_fail">Не удалось сохранить изображение</string>
+    <string name="msg_func_unavailable">Функциональность недоступна</string>
+    <string name="action_ok">ОК</string>
+    <string name="title_dialog_convert">Конвертация не удалась</string>
+    <string name="msg_convert">Конвертация не удалась.\n Вы можете попробовать
```
Comment (maik-nack) [inline app/src/main/res/values-ru/strings.xml]: Space is unnecessary

File: app/src/main/res/values-ru/strings.xml [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+        выбор, нажмите на галочку в правом верхнем углу.</string>
+    <string name="help_columns">Чтобы выбрать размер кроссворда, нажмите на
+        кнопку \'РАЗМЕР\' в правом нижнем углу.\n\nВы можете увидеть примерный
+        результат. Также вы можете увидеть над изображением текущий размер и
+        диапазон возможных размеров после преобразования. Около изображения есть
+        слайдер, который обеспечивает выбор количества столбцов. Число строк
+        рассчитывается автоматически. Выбранное изображение находится в центре
+        экрана, поверх него рисуется сетка. Чтобы зафиксировать свой выбор,
+        нажмите на галочку в правом верхнем углу.</string>
+    <string name="help_convert">После того, как вы настроили все, что вам нужно,
```
Comment (maik-nack) [inline app/src/main/res/values-ru/strings.xml]: `ё`?

**Review by maik-nack — APPROVED** [review]
GREAT JOB!!!

**Review by MaximSmolskiy — APPROVED** [review]
Looks great!

**Pull request merged**